lib/sdr_client/deposit/file_set.rb

```diff
@@ -8,7 +8,7 @@ def initialize(uploads: [], uploads_metadata: {}, files: [], label:)
         @label = label
         @files = if !uploads.empty?
                    uploads.map do |upload|
-                     File.new(file_args(upload, uploads_metadata.fetch(upload.filename, {})))
+                     File.new(**file_args(upload, uploads_metadata.fetch(upload.filename, {})).symbolize_keys)
```

Comment:
symbolize_keys isn't a method in the standard library (it's a rails monkeypatch).

Reply:
OK. Rolled my own.